chemprop/cli/predict.py

```diff
@@ -159,7 +158,6 @@ def validate_predict_args(args):
 def main(args):
     model = MPNN.load_from_checkpoint(args.checkpoint_path)
 
-    bond_messages = isinstance(model.message_passing, BondMessagePassing)
```

Comment:
bond_messages was passed through data set creation to be given to featurization. My best guess is it was never needed and accidently added by David in his large refactoring. I don't actually know what it practically means for messages to be centered on bonds or atoms, but in the v1 code this wasn't needed for featurization.

Reply:
per slack, this can be removed. this was needed in v2-alpha back when bond- vs. atom-message passing was determined by graph featurization. It's now handled inside the message passing operation.